BankSDK/GiniBankSDK/Documentation/source/Integration.md

```diff
@@ -131,17 +131,18 @@ You can also test with a real health app. Please contact us in case you don't kn
 Photo payment functionality
 ============================
 
-The Gini Bank SDK provides two integration options. A [Screen API](#screen-api) that is easy to implement and a more complex, but also more flexible [Component API](#component-api). Both APIs can access the complete functionality of the SDK.
+The SDK provides a custom `UIViewController` object, which can be presented modally. It handles the complete process from showing the onboarding until providing a UI for the analysis.
```

Comment:
[Suggestion] "UIViewController" can be presented in different ways (can be pushed as well). Modal presentation is just a particular case. Shall we write something more generic?